share/eds/byzantine/byzantine.go

```diff
@@ -45,42 +43,50 @@ func NewErrByzantine(
 	sharesWithProof := make([]*ShareWithProof, len(errByz.Shares))
 	sharesAmount := 0
 
-	errGr, ctx := errgroup.WithContext(ctx)
-	for index, share := range errByz.Shares {
-		// skip further shares if we already requested half of them, which is enough to recompute the row
-		// or col
-		if sharesAmount == len(dah.RowRoots)/2 {
-			break
-		}
+	type tempStruct struct {
```

Comment:
nit: `tempStruct` is not descriptive; what about `request` as you describe it PR description or just `shareWithIdx`

Reply:
my 2c: Could be just `result` and `resutlCh` as it collects results from parallel func calls